src/lib/helpers/ArrowHelper.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { ArrowHelper, Color, ColorRepresentation, Vector3 } from 'three'
+  import Object3DInstance from '../instances/Object3DInstance.svelte'
+
+  //self
+  export let dir:Vector3;
+  export let origin:Vector3;
+  export let length:number = 1;
```

Comment:
The [constructor](https://github.com/mrdoob/three.js/blob/860af3c012915be534f8a5c37d745e2dff339e04/src/helpers/ArrowHelper.js#L18) provides default values, so the component should not do so as the default values "management" is upstream.